### PR TITLE
Remove baseChannel and clarify channel naming constraints

### LINKED ISSUE #215: [FEATURE REQUEST] Add Parameter "in" property from OpenAPI
Following the Specification for the parameter object.
_"It SHOULD be present only when using channels with expressions (as defined by RFC 6570 section 2.2)."_

The RFC 6570 section 2.2 defines query-style as &name=value pairs within a literal query component.

So Channels following URI pattern and query-styles being defined the parameters should accept **"in": path** and **"in": query**.

for that the schema for Parameter Object should accept the "in" property, going close towards the OpenAPI specification.

### LINKED ISSUE #237: [2.0.0 REVIEW] Clarification on Channel naming
The [Channels](https://www.asyncapi.com/docs/specifications/2.0.0-rc1/#a-name-channelsobject-a-channels-object) object mentioned that Channel keys must start with a "/" or be relative to the server's `baseChannel` - This may be a bit ambiguous in situations where the `baseChannel` is not provided (defined as defaulting to "empty string"). I would thus assume the key is relative to "/" - effectively, treating it as an absolute path. Perhaps this can be clarified in this description block?

Additionally, a total nit-pick... Consider changing "base channel" to "baseChannel," perhaps? It makes searching of the page easier to find all references.

### PR DESCRIPTION
**Pull request by fmvilas**
Fixes #237.
Fixes #215.

### CODE REVIEW

File: versions/next/asyncapi.md [hunk 1]
```diff
@@ -479,14 +476,15 @@ defaultContentType: application/json
 
 #### <a name="channelsObject"></a>Channels Object
 
-Holds the relative paths to the individual channel and their operations.
-Channels starting with a slash (`/`) MUST be treated as absolute paths and therefore skipping the [`server's base channel`](#serverObjectBaseChannel). Otherwise, the final channel MUST be resolved using the [`server's base channel`](#serverObjectBaseChannel) and the current path.
+Holds the relative paths to the individual channel and their operations. Channel paths are relative to servers.
+
+Channels are also known as "topics", "routing keys", "event types" or "paths".
```
Comment (WaleedAshraf) [inline versions/next/asyncapi.md]: good to have this explanation 👍

**Comment by WaleedAshraf** [comment]
Where do update the actual schema definition w.r.t to these changes?

**Comment by WaleedAshraf** [comment]
I think we need to remove this also: https://github.com/asyncapi/asyncapi/blob/master/versions/next/schema.json#L193

**Comment by fmvilas** [comment]
You're right. It's missing schema changes. Thanks for pointing out.

**Comment by WaleedAshraf** [comment]
LGTM 👍

**Pull request merged**